tests/lib/rules/prefer-numeric-literals.js

```diff
@@ -49,9 +50,17 @@ ruleTester.run("prefer-numeric-literals", rule, {
             code: "parseInt('11', 16n);",
             parserOptions: { ecmaVersion: 2020 }
         },
+        {
+            code: "parseInt(`11`, 16n);",
+            parserOptions: { ecmaVersion: 2020 }
+        },
         {
             code: "parseInt(1n, 2);",
             parserOptions: { ecmaVersion: 2020 }
+        },
+        {
+            code: "parseInt(`11${foo}`, 2);",
+            parserOptions: { ecmaVersion: 2016 }
```

Comment:
I think that `ecmaVersion: 2016` isn't necessary here?

Reply:
@mdjermanovic it is unnecessary. Thanks! I changed it